code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -201,6 +201,11 @@
 		M.ExtinguishMob()
 	..()
 
+/datum/reagent/water/on_mob_life(mob/living/carbon/M)
+	. = ..()
+	if(M.blood_volume)
+		M.blood_volume += 0.1 // water is good for you!
```

Comment:
it's slime(person) time, boys

also, uh, keep in mind that if your blood_volume goes too high, you explode into a pile of gibs. might wanna add a check to make sure that this can't bring you above that threshold.